### PR TITLE
exclude mock_kraken from sonar code coverage

### PR DESCRIPTION
**Pull request by pbench**
to avoid false alarm from sonarcloud analysis like https://sonarcloud.io/summary/new_code?id=Hove_navitia&pullRequest=3781

### CODE REVIEW

**Review by woshilapin — APPROVED** [review]

File: sonar-project.properties [hunk 1]
```diff
@@ -14,7 +14,7 @@ sonar.cfamily.build-wrapper-output=build-wrapper-output-dir
 sonar.cfamily.gcov.reportsPath=.
 sonar.coverageReportPaths=cov.xml
 sonar.python.coverage.reportPaths=jormungandr_serpy_integration_coverage.xml,jormungandr_coverage_serpy.xml,tyr_integration_coverage.xml
-sonar.coverage.exclusions=source/jormungandr/tests/**,source/tyr/tests/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**
+sonar.coverage.exclusions=source/jormungandr/tests/**,source/tyr/tests/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**,source/tests/**
```
Comment (woshilapin) [inline sonar-project.properties]:
```suggestion
sonar.coverage.exclusions=source/**/tests/**,source/benchmark/**,source/cities/**,source/eitri/**,source/log_analyzer/**,source/monitor/**,source/scripts/**,source/sql/**,source/third_party/**,source/tools/**
```

Comment (pbench) [reply sonar-project.properties]: are you sure this syntax is allowed ? I can't find a documentation explaining what is allowed or not here

Comment (woshilapin) [reply sonar-project.properties]: It looked like standard bash glob so I assumed it was but you might be right, I may have extrapolated :shrug: 

That said, it seems [to be possible](https://docs.sonarqube.org/latest/analysis/test-coverage/test-coverage-parameters/).

Comment (pbench) [reply sonar-project.properties]: allright, let's try it !

**Pull request merged**